 * Serving Flask app 'dashboard'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.2.114:5001
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 918-818-938
Traceback (most recent call last):
  File "/Users/ashishtandon/Github/Base Code/backend/OpenPolicyAshBack/dashboard.py", line 624, in <module>
    app.run(host='0.0.0.0', port=5001, debug=True)
    ~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ashishtandon/Github/Base Code/backend/venv/lib/python3.13/site-packages/flask/app.py", line 662, in run
    run_simple(t.cast(str, host), port, self, **options)
    ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ashishtandon/Github/Base Code/backend/venv/lib/python3.13/site-packages/werkzeug/serving.py", line 1115, in run_simple
    run_with_reloader(
    ~~~~~~~~~~~~~~~~~^
        srv.serve_forever,
        ^^^^^^^^^^^^^^^^^^
    ...<3 lines>...
        reloader_type=reloader_type,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/ashishtandon/Github/Base Code/backend/venv/lib/python3.13/site-packages/werkzeug/_reloader.py", line 459, in run_with_reloader
    ensure_echo_on()
    ~~~~~~~~~~~~~~^^
  File "/Users/ashishtandon/Github/Base Code/backend/venv/lib/python3.13/site-packages/werkzeug/_reloader.py", line 439, in ensure_echo_on
    termios.tcsetattr(sys.stdin, termios.TCSANOW, attributes)
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
termios.error: (4, 'Interrupted system call')
